summary_workflows/identification/I5_benchmark/I5_benchmark_identification.md

```diff
@@ -0,0 +1,105 @@
+# I5 benchmark specification
+
+## Synopsis
+
+Benchmark to compare RNAseq-based site identification with ground truth, i.e. the poly(A) sites of transcripts used to generate simulated RNA-seq data, for several different distance thresholds.
```

Comment:
Maybe it could be rephrased to make it clear that RNAseq data is simulated? e.g.

> Benchmark to compare **PAS identification from simulated RNAseq data** with ground truth ...